warehouse/transformer/transformer_benchmark_test.go

```diff
@@ -10,19 +10,18 @@ import (
 	"github.com/rudderlabs/rudder-go-kit/config"
 	"github.com/rudderlabs/rudder-go-kit/logger"
 	"github.com/rudderlabs/rudder-go-kit/stats"
+
 	"github.com/rudderlabs/rudder-server/jsonrs"
 	"github.com/rudderlabs/rudder-server/processor/types"
 )
 
 /*
-Benchmark_Transformer/Identify_(POSTGRES)_-_1000_events
-Benchmark_Transformer/Identify_(POSTGRES)_-_1000_events-12         	       9	 112701398 ns/op
```

Comment:
Earlier we had 2 `ns/op`, now only 1. What changed?

Reply:
We do have only 1 Benchmark within the test. This one is not required.